src/Diagnostics.DataProviders/DataProviderContext.cs

```diff
@@ -13,6 +13,7 @@ public class DataProviderContext
         public IWawsObserverTokenService WawsObserverTokenService { get; private set; }
         public ISupportBayApiObserverTokenService SupportBayApiObserverTokenService { get; private set; }
         public IKustoHeartBeatService KustoHeartBeatService { get; private set; }
+        public string GeomasterName { get; private set; }
```

Comment:
nit : is this the geomasterEndpoint or just the name of the cloud service?  If its an endpoint, lets call it out explicitly in the variable.

Reply:
It's just a DNS name. I initially thought I wanted to use an endpoint name like "https://...." but found that it's a bit difficult to get the CSM port number there. So I just ended up with the DNS name.